pkg/option/option.go

```diff
@@ -137,6 +137,9 @@ type Option struct {
 	// to learn about the system resources.
 	PathOverrides PathOverrides
 
+	// LoopDevices optionally allows reporting disk/partitions that are loop devices
+	LoopDevices *bool
```

Comment:
I'm tempted to say let's get rid of this option and just always process loop devices, and let callers filter out any results the want... @fromanirh what do you think?

Reply:
@Itxaka yeah, just remove this config option... if there is a huge demand for it, we can always add an option in the future.

Reply:
Hey! I didn't had time to think this change (overall PR not this specific bit) deep enough. Will do next week. In general I tend to agree to add to the API surface gradually.